templates/CRM/common/TabHeader.tpl

```diff
@@ -37,3 +37,4 @@
   {/if}
   <div class="clear"></div>
 </div> {* crm-content-block ends here *}
+{include file="CRM/common/TabSelected.tpl" defaultTab="settings"}
```

Comment:
Haven't we lost some nuance on defaultTab here?

Reply:
In theory, but in practise they all default to the first tab so the behaviour before/after is the same. Also to add some fun `selectedChild` can and normally is set on the PHP side so the default is taken from there.

Reply:
This currently breaks the default loading of tabs when contribution page is accessed from the action links. Eg from https://dmaster.demo.civicrm.org/civicrm/admin/contribute?reset=1 -> select `Configure` > `Membership Settings` should navigate to the Membership tab directly but instead is navigated to the first tab of the page(Settings).

Can be replicated by simply clicking on https://dmaster.demo.civicrm.org/civicrm/admin/contribute/membership?reset=1&action=update&id=1  (opens settings page instead of membership). Do we aim to support `selectedChild` parameter in this URL?

Reply:
@jitendrapurohit did you open a gitlab?

Reply:
gitlab is created now - https://lab.civicrm.org/dev/core/-/issues/1738

Reply:
@mattwire imo - maybe we should just revert this line for the fix?